demos/mqtt/mqtt_demo_basic_tls/mqtt_demo_basic_tls.c

```diff
@@ -209,11 +209,24 @@ static uint16_t globalUnsubscribePacketIdentifier = 0U;
  */
 static PublishPackets_t outgoingPublishPackets[ MAX_OUTGOING_PUBLISHES ] = { 0 };
 
+/**
+ * @brief Array to keep subscription topics.
+ * Used to re-subscribe to topics that failed initial subscription attempts.
+ */
+static MQTTSubscribeInfo_t pSubscriptionList[ 1 ] = { 0 };
```

Comment:
We use the `p` prefix only for pointer types
```suggestion
static MQTTSubscribeInfo_t globalSubscriptionList[ 1 ] = { 0 };
```